### PR TITLE
Fixes #7071

### LINKED ISSUE #7071: Desk bells can sometimes be picked up by bags???
Both ring when clicked, but only one can be picked up by the bag.
#4666: The reopening

### PR DESCRIPTION
**Pull request by ghost**
Fixes #7071

### CODE REVIEW

File: code/game/objects/items/devices/deskbell.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 	var/wrenching = 0
 
 /obj/item/device/deskbell/attackby(obj/item/W, mob/user)
-	if(istype(W,/obj/item/weapon/wrench))
+	if(istype(W,/obj/item/weapon/wrench) && src.loc == get_turf(src))
```
Comment (clusterfack) [inline code/game/objects/items/devices/deskbell.dm]: This is just too silly for me, make this isturf(loc)

File: code/game/objects/items/devices/deskbell.dm [hunk 2]
```diff
@@ -77,7 +77,10 @@
 	return
 
 /obj/item/device/deskbell/attack_hand(var/mob/user)
-	ring()
+	if(anchored)
+		ring()
+	else
+		..()
```
Comment (clusterfack) [inline code/game/objects/items/devices/deskbell.dm]: Also this isn't necessary

Comment (ghost) [inline code/game/objects/items/devices/deskbell.dm]: Unless you want people ringing deskbells in their backpack, which really isn't intended, and having to click-drag the deskbell to themselves to pick it up which is unintuitive and silly, (actually the cause of the issue is assuming you can't pick it up) this is necessary.

Comment (clusterfack) [inline code/game/objects/items/devices/deskbell.dm]: Not this entire section, the else ..()

Comment (ghost) [inline code/game/objects/items/devices/deskbell.dm]: does it automatically else ..() ? There's no ..() elsewhere.

Comment (clusterfack) [inline code/game/objects/items/devices/deskbell.dm]: Does it need it?

Comment (ghost) [inline code/game/objects/items/devices/deskbell.dm]: I.. I don't know. I assumed to pick it up it needed a ..()

Comment (clusterfack) [inline code/game/objects/items/devices/deskbell.dm]: Pretty silly desu

Comment (gbasood) [inline code/game/objects/items/devices/deskbell.dm]: Why would you need to pick it up if it's anchored

Comment (ghost) [inline code/game/objects/items/devices/deskbell.dm]: > if anchored blah blah else ..()

it wouldn't be anchored

**Pull request merged**